ui/src/main/webapp/src/app/shared/confirmation-modal.component.ts

```diff
@@ -16,11 +16,16 @@ import * as $ from "jquery";
                 </h4>
             </div>
             <div class="modal-body">
-                {{body}}
+                <p>{{body}}</p>
+                <p [style.display]="confirmPhrase ? 'block' : 'none'">
+                    <input #confirmInput style="margin: 1ex 1em" type="text"
+                       (keyup)="yes.disabled = (confirmPhrase && ($event.target.value.trim().toLowerCase() != confirmPhrase.trim().toLowerCase()))" >
```

Comment:
I would prefer having this code in class instead of template.
(I'm quite surprised you put it here)